src/pages/cli/project/command-hooks.mdx

```diff
@@ -46,15 +46,15 @@ Minimum CLI version requirement met.
 ```
 
```

Comment:
Please note: Hooks need to be checked into source control in order to be made available across machines. `amplify pull` will **not** re-create the `hooks` folder automatically as these are considered changes that need to be committed to the repository.